.github/workflows/backup-cognito-users.yaml

```diff
@@ -0,0 +1,44 @@
+name: Create backup of cognito users
+on:
+  workflow_dispatch:
+env:
+  region: 'eu-west-1'
+  backups-bucket: 'biomage-backups-production'
+
+jobs:
+  backup-users:
+    name: create backup of cognito users
+    runs-on: ubuntu-20.04
+    steps:
+      - id: setup-node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - id: install
+        name: Install dependencies
+        run: |-
+          npm install -g cognito-backup-restore
+          sudo apt-get install jq
+
+      - id: set-up-creds
+        name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.region }}
+          
+      - id: backup-users
+        name: create backup json of cognito users
+        run: |-
+          userPoolId=$(aws cognito-idp list-user-pools --max-results 20 | jq -c '.UserPools[] | select(.Name | test("production")) | .Id')
```

Comment:
I would suggest to test that the name is exactly ``biomage-user-pool-case-insensitive-${k8sEnv}`` as we do in `src/config/default-config.js` in the API. We were testing only whether the pool name contained `production` and that broke during the migration to the new pools (which also contained the word production)